web/src/app/util/Search.js

```diff
@@ -73,12 +72,14 @@ export default function Search(props) {
 
   // When typing, we setup a debounce before updating the URL.
   useEffect(() => {
+    const setSearchParam = value => dispatch(setURLParam('search', value))
+
     const t = setTimeout(() => {
       setSearchParam(search)
     }, DEBOUNCE_DELAY)
 
     return () => clearTimeout(t)
-  }, [search])
+  }, [dispatch, search])
```

Comment:
We only want this to trigger on `search` change, not when the dispatch function changes